### DISCUSSION TITLE
Tasmota SetOption114 1 bevavior in Home Assistant

### DISCUSSION THREAD

**Discussion by boardlord1** [arendst/Tasmota#18646] [Q&A]
Hi guys, quick question. If I detach the switch in a Shelly 1 with Tasmota using SetOption114, how is it represented in Home Assistant? Do both the switch and the relay show up as toggles? Or do I need to create template switches that toggle based on MQTT messages to get the relay and switch toggles?

**Comment by bernardesarthur** [comment]
Hi bro, when you detach switch from your relay, binary sensors are created on your Home Assistant and you can use to everything you want.
<img width="914" alt="Captura de Tela 2024-10-30 às 07 23 55" src="https://github.com/user-attachments/assets/087bfa1d-3d7e-4470-9988-cd2ed415b00b">
